repos/system_upgrade/el7toel8/files/rhel_upgrade.py

```diff
@@ -55,6 +55,32 @@ def pre_configure(self):
         self.base.conf.releasever = self.plugin_data['dnf_conf']['releasever']
 
     def configure(self):
+        def read_aws_region(repo):
```

Comment:
These functions don't seem to be needed in here, you could have them normal member functions, no?

Reply:
what would be the benefit? I mean like I am indifferent here but I do not want to change it just for a sake of change...

Reply:
I guess the reason is that you can't test these inner functions @Rezney 
However the re is a nice aricle on use cases for inner functions https://realpython.com/inner-functions-what-are-they-good-for/

Reply:
moved...